code/datums/components/material_container.dm

```diff
@@ -532,6 +532,7 @@
 			"name" = material.name,
 			"ref" = REF(material),
 			"amount" = amount,
+			"SHEET_MATERIAL_AMOUNT" = SHEET_MATERIAL_AMOUNT,
```

Comment:
This is fine too, but to clarify this is what I meant

```
/datum/component/material_container/ui_static_data(mob/user)
	var/list/data = list()
	data["sheet_amount"] = SHEET_MATERIAL_AMOUNT
	return data
```

You can look at ``coopIndex`` and ``fundingIndex`` for current examples of it in-code.
Sorry I couldn't be more clear earlier.